JuliaGPU/session2/Gauri/q1.jl

```diff
@@ -0,0 +1,30 @@
+#Sorting a random array of 2^20 elements using bubble sort and
+
+#a = rand(2^3)
+#println("Random array a before sorting: $a")
+#println("a after sorting:")
+function sortfunc(a)
```

Comment:
For 2^20 elements, using bubble sort to sort is very inefficient. So you will need to implement an efficient sorting algorithm like merge sort or quick sort